docs/en/05_Contributing/04_Release_Process.md

```diff
@@ -31,18 +31,27 @@ announced on the ["releases" forum category](https://forum.silverstripe.org/c/re
 
 SilverStripe follows [Semantic Versioning](http://semver.org).
 
-## Supported versions
+## Supported versions {#supported-versions}
 
-Our version support is outlined on [silverstripe.org/roadmap](http://silverstripe.org/roadmap).
 At any point in time, the core development team will support a set of releases to varying levels:
 
-*  The current *master* will get new features, bug fixes and API changes that might require major refactoring before going
-into a release. At the moment, bugfixing and feature development might happen on the current major release branch (e.g. *4*), to be
-merged forward to master regularly.
-*  Applicable bugfixes on master will also be merged back to the last major release branch, to be released as the next
-patch release
-*  Security fixes will be applied to the current master,
-   as well as patch releases for SilverStripe versions in "active development", "full support" or "limited support".
+ * The status of major releases is determined by the [roadmap]((http://silverstripe.org/roadmap))
+ * Minor releases of major releases in "active development" or "full support" are released roughly every three months, and their End-of-Life (EOL) is announced at least six months in advance
+ * The latest minor release is supported as long as the underlying major release
+ * API changes and major new features are applied to the master branch, to be included in the next major release
+ * New APIs can be applied to the current minor release of major releases in "active development", but should usually be marked as "internal" APIs until they're considered stable
+ * Enhancements are applied to the latest minor release of major releases in "active development"
+ * Non-critical bugfixes are applied to all supported minor releases of major releases in "active development" or "full support"
+ * Critical bugfixes and security fixes are applied to the all minor releases of major releases in "active development", "full support" or "limited support"
+ * Any patches applied to older minor releases are merged up regularly to newer minor releases (in the same major release)
+ * Any patches applied to older major releases are merged up regularly to newer major releases
```

Comment:
This is pretty tricky to achieve, especially with framework splitting off into other modules in SS4

Reply:
"merged up" could mean manually. But in general, if a problem is fixed in 3.x, and it exists in 4.x as well, we need to figure out how to fix it there as well. That might be as straightforward as a merge-up, a cherry pick with some class rename, or in some cases a different approach.